packages/web-components/fast-foundation/src/calendar/calendar.template.ts

```diff
@@ -0,0 +1,77 @@
+import { html, repeat } from "@microsoft/fast-element";
+import type { ViewTemplate } from "@microsoft/fast-element";
+import { endTemplate, startTemplate } from "../patterns/start-end";
+import type { Calendar, CalendarDateInfo } from "./calendar";
+
+/**
+ * A basic Calendar title template that includes the month and year
+ * @public
+ */
+export const CalendarTitleTemplate: ViewTemplate<Calendar> = html`
+    <slot name="title">
+        <div class="title" part="title">
+            <span part="month">${x => x.getMonth()}</span>
+            <span part="year">${x => x.getYear()}</span>
+        </div>
+    </slot>
+`;
+
+/**
+ * Calendar weekday label template
+ * @public
+ */
+export const CalendarWeekdayTemplate: ViewTemplate<Calendar> = html`
+    <div class="week-day" part="week-day">${x => x}</div>
+`;
+
+/**
+ * A calendar day template
+ * @public
+ */
+export const CalendarDayTemplate: ViewTemplate<CalendarDateInfo> = html`
+    <div
+        part="day"
+        aria-label="${(x, c) =>
+            ("getMonth" in c.parent ? c : c.parentContext).parent.getMonth(
+                x.month
+            )} ${x => x.day}"
```

Comment:
@scomea added the month and day example: "July 26" as the aria label for each cell.

Reply:
Ran into some issues with aria-labels for abbreviated weekday names. I added labels but seems to read off the abbreviated if it's an actual word. Reads "sat" for Sat but reads "Friday" for Fri. I might need to play around with the abbr attribute.